chart/templates/csi-daemonset.yaml

```diff
@@ -1,128 +1,8 @@
-        imagePullPolicy: {{ .Values.mayastorImagePullPolicy }}
-        # we need privileged because we mount filesystems and use mknod
-        securityContext:
-          privileged: true
-        env:
-        - name: MY_NODE_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: spec.nodeName
-        - name: MY_POD_IP
-          valueFrom:
-            fieldRef:
-              fieldPath: status.podIP
-        - name: RUST_BACKTRACE
-          value: "1"{{ if .Values.moac }}
-        - name: MOAC
-          value: "true"{{ end }}
-        args:
-        - "--csi-socket=/csi/csi.sock"
-        - "--node-name=$(MY_NODE_NAME)"
-        - "--grpc-endpoint=$(MY_POD_IP):10199"{{ if .Values.csi.nvme.io_timeout_enabled }}
-        - "--nvme-core-io-timeout={{ .Values.csi.nvme.io_timeout }}"{{ end }}
-        - "-v"
-        command:
-        - mayastor-csi
-        volumeMounts:
-        - name: device
-          mountPath: /dev
-        - name: sys
-          mountPath: /sys
-        - name: run-udev
-          mountPath: /run/udev
-        - name: host-root
-          mountPath: /host
-        - name: plugin-dir
-          mountPath: /csi
-        - name: kubelet-dir
-          mountPath: /var/lib/kubelet
-          mountPropagation: "Bidirectional"
-        resources:
-          limits:
-            cpu: "100m"
-            memory: "50Mi"
-          requests:
-            cpu: "100m"
-            memory: "50Mi"
-      - name: csi-driver-registrar
-        image: quay.io/k8scsi/csi-node-driver-registrar:v2.1.0
-        args:
-        - "--csi-address=/csi/csi.sock"
-        - "--kubelet-registration-path=/var/lib/kubelet/plugins/mayastor.openebs.io/csi.sock"
-        volumeMounts:
-        - name: plugin-dir
-          mountPath: /csi
-        - name: registration-dir
-          mountPath: /registration
-        resources:
-          limits:
-            cpu: "100m"
-            memory: "50Mi"
-          requests:
-            cpu: "100m"
-            memory: "50Mi"
-        # Mayastor node plugin gRPC server
-        ports:
-        - containerPort: 10199
-          protocol: TCP
-          name: mayastor-node
-      volumes:
-      - name: device
-        hostPath:
-          path: /dev
-          type: Directory
-      - name: sys
-        hostPath:
-          path: /sys
-          type: Directory
-      - name: run-udev
-        hostPath:
-          path: /run/udev
-          type: Directory
-      - name: host-root
-        hostPath:
-          path: /
-          type: Directory
-      - name: registration-dir
-        hostPath:
-          path: /var/lib/kubelet/plugins_registry/
-          type: Directory
-      - name: plugin-dir
-        hostPath:
-          path: /var/lib/kubelet/plugins/mayastor.openebs.io/
-          type: DirectoryOrCreate
-      - name: kubelet-dir
-        hostPath:
-          path: /var/lib/kubelet
-          type: Directory
+  name: csi-daemonset-hack
```

Comment:
Why are we doing this? Cant the daemonset be removed from here?

Reply:
Temporary not to break e2e pipeline